pkg/database/database.go

```diff
@@ -190,9 +191,9 @@ func (db *Database) getOwnedPodsUuids(ctx context.Context, ownersUuids, podUuids
 
 func (db *Database) performResourceQuery(ctx context.Context, pq *paramQuery) ([]*unstructured.Unstructured, int64, string, error) {
 	type resourceFields struct {
-		Date  string       `json:"created_at"`
-		Id    int64        `json:"id"`
-		Bytes sql.RawBytes `json:"data"`
+		Date     string `db:"created_at"`
+		Id       int64  `db:"id"`
+		Resource string `db:"data"`
```

Comment:
Since `Resource` needs to be converted to `[]byte` anyway I think would make sense to just get `Resource` as `[]byte`

```suggestion
		Resource []byte `db:"data"`
```

Reply:
So, there is an issue regarding the retrieval of bytes because they are kind of "managed" by the driver and sometimes could be overwritten from row to row retrieval depending on the driver.

Please take a look at the links I posted on "Notes for Reviewers".

Reply:
my understanding is that the sql driver treats `[]bytes` and `RawBytes` differently. From the docs for `row.Scan`:

> If a dest argument has type *[]byte, Scan saves in that argument a copy of the corresponding data. The copy is owned by the caller and can be modified and held indefinitely

Reply:
oh! then let me check it out then! I thought that they were the same as `RawBytes` is an alias of `[]bytes` thanks for pointing it out :)

Reply:
right! it works! Updated :)

Reply:
I've been confused by this too

```
type RawBytes []byte // type declaration. RawBytes and []byte are treated as different types and explicit conversion is required between the two types

type RawBytes = []byte // type alias. RawBytes is just another way to refer to []byte. No conversion is required because RawBytes is []byte
```

[This blog post](https://go.dev/blog/alias-names) from the Go devs help me understand the difference better

```
type RawBy